src/common/options/osd.yaml.in

```diff
@@ -440,17 +440,30 @@ options:
   type: float
   level: advanced
   desc: Deep scrub each PG (i.e., verify data checksums) at least this often
-  fmt_desc: The interval for "deep" scrubbing (fully reading all data). The
-    ``osd_scrub_load_threshold`` does not affect this setting.
+  fmt_desc: The interval for "deep" scrubbing (fully reading all data).
   default: 7_day
   with_legacy: true
+- name: osd_deep_scrub_interval_cv
+  type: int
+  level: advanced
+  desc: determining the amount of variation in the deep scrub interval
+  long_desc: deep scrub intervals are varied by a random amount to prevent
+    stampedes. This parameter determines the amount of variation.
+    Technically - osd_deep_scrub_interval_cv is the coefficient of variation for the deep scrub interval.
+  fmt_desc: The coefficient of variation for the deep scrub interval. The next deep
+    scrub for a PG is scheduled osd_deep_scrub_interval after the last deep scrub in average. The
+    actual time is randomized to a normal distribution with a standard deviation of
+    osd_deep_scrub_interval * osd_deep_scrub_interval_cv (clamped to within 2 standard deviations).
```

Comment:
That looks wrong -- the values are default 20 and max 40.  If you mean this to be a percentage, include the .01.  I'd actually prefer you simply represent this ratio between 0 and 1 and skip the percentage bit entirely.

Reply:
I was trying to avoid inventing a new term to specify the size of the randomized range in a way that is not dependent on the average. And - I do not need to: the CV is the accepted term for this. And the CV is expressed in percentages.
(Note that I do not specify that it's percentages - but that will be obvious to anyone who is familiar with statistics.)

Reply:
Specify that it is an integer percentage.